master/getting-started/docker/tutorials/security-using-docker-labels-and-calico-policy.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Security using Docker Labels and Calico Policy
+---
+
+## Background
+
+With Calico as a Docker network plugin, Calico can be configured to extract the
+labels on a container and apply them to the workload endpoint for use with Calico
+[policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy).
+All traffic must be explicitly allowed through configuration of the globally
+defined policy which uses selectors to determine which subset of the policy is
+applied to each container based on their labels.  This approach provides a
+powerful way to group together all of your network Policy, makes it easy to
+reuse policy in different networks, and makes it easier to define policy that
+extends across different orchestration systems that use Calico.
+
+When Calico is configured to use container labels, profiles are not created and 
+have no impact on any container traffic.
+
+## Enabling Docker Networking Container Labels Policy
+
+To enable the container labels Policy with Docker Networking the flag
+`--use-docker-networking-container-labels` must be passed when starting
+calico/node with the `calicoctl node run` command.  All calico/node instances
```

Comment:
Don't we have a way to set global Calico settings in etcd? I think we have it for felix, not sure if calico/node startup supports it though